installing_on_kubernetes/_topics/image_pull_secret.md

```diff
@@ -0,0 +1,5 @@
+## Configuring an image pull secret
+
+If authentication is required in order to pull the images, a secret containing the credentials needs to be created.
+If your secret is named `image-pull-secret`, no additional changes are required to the custom resource.
+If you use a different name for the secret, that will need to be defined in the custom resource as `imagePullSecret: your-secret-name`
```

Comment:
super minor, but can we structure this similar to the other docs like custom_oidc_ca_certificate.md where it has an oc command and then an example secret?